.github/workflows/e2e.yml

```diff
@@ -0,0 +1,76 @@
+name: CI E2E
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: 
+      - main
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  e2e-basic-install:
+    runs-on: ubuntu-20.04
+    env:
+      built_image: "rte-operator:ci" # Arbitrary name
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # Fetch all history for all tags and branches
+
+    - name: Set up golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: Verify modules
+      run: go mod verify
+
+    - name: build test binary
+      run: |
+        make binary-e2e
+
+    - name: Build image
+      run: |
+        docker build . -t ${built_image}
+
+    - name: Create K8s Kind Cluster
+      run: |
+        kind create cluster --config=hack/kind-config-e2e-no-registry.yaml --image kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
```

Comment:
from where is coming kind binary? Can we use a newer version for the kindest/node?

Reply:
It's part of the base image (added comment to act as reminder).
Kind still recommends this image: https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1
once the lane is stable I'd like to try the 1.22 image.